Sources/SafeDICore/Models/TypeDescription.swift

```diff
@@ -249,6 +257,8 @@ public enum TypeDescription: Codable, Hashable, Comparable, Sendable {
         case typeDescription
         /// The value for this key is the associated value of type [TypeDescription]
         case typeDescriptions
+        /// The value for this key is the associated value of type Set<TypeDescription>
+        case unorderedTypeDescriptions
```

Comment:
I'm not convinced this is a necessary addition. While the extra specificity is nice... the underlying json representations of a `Set` and `Array` are identical (aside from ordering).

So my options are:

1. Introduce this new key
2. Update the documentation on `case typeDescriptions` to state that the value here is either an array or set.

I'm currently leaning towards the second option.

Reply:
Yeah alright talked myself into it and pushed up 8afe45ea537d76a0c0aa2b80cf31f0ab06d8a6cb. Open to feedback, and could be convinced to revert.